ui/webpack.config.js

```diff
@@ -0,0 +1,69 @@
+const path = require('path');
+
+// check/detect if a production build (or not) via the webpack -p flag
+const IS_PRODUCTION = process.argv.indexOf('-p') != -1;
+
+const parentDir = path.join(__dirname, './');
+const PUBLIC_DIR = path.resolve(__dirname, './public/');
+const BUILD_DIR = path.resolve(__dirname, './dist')
+
+const htmlPlugin = require('html-webpack-plugin');
+
+const htmlPluginConfiguration = {
+  filename: 'index.html',
+  template: PUBLIC_DIR + '/index.html',
+  title: 'Kafka Java VertX Starter UI'
+};
+
+const pluginSet = [
+  new htmlPlugin(htmlPluginConfiguration)
+]
+
+module.exports = {
+  mode: IS_PRODUCTION ? 'production' : 'development',
```

Comment:
would be curious to know what the bundles sizes are. Ie do we need to worry about gzipping/splitting into seperate bundles.

Also, would be interesting to see the effect of `sideEffects` https://webpack.js.org/guides/tree-shaking/ (mentioned here as this involves bundles, but sideEffects goes into the package json

Reply:
Current bundle size is 134Kb, it has already gone up to 1mb on my branch with all the extra listing modules.

Me too! Do you think this is worth trying to add once we have more code so we can see how it behaves?

Reply:
I would give it (sideEffects) a go now and see what happens (we have version of Webpack which use it, so why not!). I also have added a note re checking bundle size in future. Gzipping etc can follow in a seperate issue if needed.

Reply:
Will be looked at seperatley under #7